SwiftDynamicFramework/SwiftDynamicLibrary.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SwiftDynamicLibrary.swift
```

Comment:
Swift code in a dynamic framework with mixed code.  This is working as expected.